dolfyn/rotate/api.py

```diff
@@ -22,7 +22,7 @@
     'rdi': r_rdi}
 
 
-def rotate2(ds, out_frame='earth'):
+def rotate2(ds, out_frame='earth', inplace=True):
```

Comment:
I'm torn here... On the one hand, I feel like we want `inplace=False` for these functions. I generally expect functions to return something to me. But we definitely want `inplace=True` for the methods. So maybe this inconsistency between those two places will create confusion?

Reply:
After thinking about this more, I think it's best to leave these consistent with the methods. So `inplace=True` is the right way to go here.